flink-core/src/main/java/org/apache/flink/api/common/state/StateTtlConfig.java

```diff
@@ -282,7 +282,7 @@ public Builder cleanupIncrementally(
 		 * https://github.com/facebook/rocksdb/blob/master/include/rocksdb/compaction_filter.h#L69
 		 * It means that the TTL filter should be tested for List state taking into account this caveat.
 		 *
-		 * @deprecated Use more general configuration method {@link #cleanupInBackground()} instead
+		 * @deprecated
```

Comment:
As we introduced `cleanupInBackground` to deprecate the function `cleanupInRocksdbCompactFilter`, now, we removed `cleanupInBackground`, do we need to remove the function `cleanupInRocksdbCompactFilter` also?

Reply:
This should be addressed in #12307.